src/components/popover/popover.ts

```diff
@@ -66,7 +66,9 @@ export class MdlPopoverComponent implements AfterViewInit {
     private hideAllPopovers() {
       let nodeList = document.querySelectorAll('.mdl-popover.is-visible');
       for(let i=0; i < nodeList.length;++i) {
-        nodeList[i].dispatchEvent(new Event('hide'));
+          let event = document.createEvent('CustomEvent');
+          event.initCustomEvent('hide', false, false, {});
```

Comment:
What about https://msdn.microsoft.com/en-us/library/ff975299(v=vs.85).aspx

> Important

  As of Microsoft Edge, the createEvent()/initEvent() constructor pattern for synthetic events is deprecated. See the Synthetic Events page for more information.

and

https://developer.mozilla.org/en/docs/Web/API/Event/initEvent

> Do not use this method anymore as it is deprecated.
> 
> Instead use specific event constructors, like Event(). The page on Creating and triggering events gives more information about the way to use these.
>